dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/ProcessDefinitionController.java

```diff
@@ -637,25 +637,25 @@ public Result batchDeleteProcessDefinitionByCodes(@ApiIgnore @RequestAttribute(v
                                                       @ApiParam(name = "projectCode", value = "PROJECT_CODE", required = true) @PathVariable long projectCode,
                                                       @RequestParam("codes") String codes) {
         Map<String, Object> result = new HashMap<>();
-        List<String> deleteFailedCodeList = new ArrayList<>();
+        Set<String> deleteFailedCodeSet = new HashSet<>();
```

Comment:
If this pr is just aimed to solve the duplicated error message.  Then it's ok to me.

Reply:
Yes, only the administrator user can delete the workflow created by other users, and the delete operation is an important operation. The one created by oneself can be deleted or the administrator can delete it. I think this logic is correct at present, but the printing There is duplication of information, so I made a bug fix for the duplicate error information

Reply:
I think we should add this administrator condition to the batch delete docs. So that we can prevent misunstanding when users use this function. @SnowMoon-Dev

Reply:
> If this pr is just aimed to solve the duplicated error message. Then it's ok to me.

ok, thanks

Reply:
Good idea, I'll add this description to the documentation later